### ISSUE TITLE
Docs need search functionality

### ISSUE THREAD

**Issue by twheys** [kayak/redux-data-model#5]
Please add a search field to the docs. This would make it significantly easier to find documentation about specific syntax between the tutorial, concepts, and other pages.

**Comment by x8lucas8x** [comment]
Makes sense. Docusaurus can actually be integrated with algolia for that.
